core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/JavaConsole.java

```diff
@@ -292,9 +291,9 @@ public void actionPerformed(ActionEvent e) {
             @Override
             public void actionPerformed(ActionEvent e) {
                 printMemoryInfo();
-                LOG.info(R("CONSOLErunningFinalizers"));
+                LOG.info("CONSOLErunningFinalizers");
```

Comment:
Hi @medoror 
this change is not enough since now `CONSOLErunningFinalizers` would be printed in the LOG. This is not really an information that would help since `CONSOLErunningFinalizers` is the name of the parameter that should be printed in the log. When having a look at the Messages.properties file in the repo you can find all the english translations for the parameters. In this case you can find the following line:
`CONSOLErunningFinalizers = Running finalization....
`
Based on this the code should be refactored to:
`LOG.info("Running finalization....");`

The same pattern must be used for all changed where the `R` method has been removed.

Reply:
Thanks for the clarification. I was not aware about message.properties. Please see my latest commit. I pulled from the upstream and rebased my commits. Let me know if I am missing anything